examples/jsonplaceholder.yaml

```diff
@@ -0,0 +1,12 @@
+# yaml-language-server: $schema=../generated/.tailcallrc.schema.json
+
+server:
+  port: 8000
```

Comment:
Can we drop `server` `upstream` and `telemetry` from the `tailcallrc.graphql` also?

Reply:
yes, dropped from the generated schema